src/technologies/r.json

```diff
@@ -633,9 +633,9 @@
     "icon": "React Router.svg",
     "implies": "React",
     "oss": true,
-    "scriptSrc": [
-      "(?:/react-router(@|/)([\\d.]+)(?:/[a-z]+)?)?/react-router(?:\\.min)?\\.js\\;version:\\2"
-    ],
+    "js": {
+      "__reactRouterVersion": "([\\d\\.]+)\\;version:\\1"
```

Comment:
This was only included in version 6. Any reason not to include the other detection too to pick up older versions?

Then again the number of sites found with this is only a couple of hundred so not massive deal to lose them.

Reply:
Nope that all makes sense to me. Just making sure you’d considered that. Let’s merge.